include/finufft/defs.h

```diff
@@ -225,8 +231,12 @@ typedef struct FINUFFT_PLAN_S { // the main plan object, fully C++
   FLT *phiHat2;        // " y-axis.
   FLT *phiHat3;        // " z-axis.
 
-  FFTW_CPX *fwBatch;   // (batches of) fine grid(s) for FFTW to plan
+#ifdef FINUFFT_USE_DUCC0
```

Comment:
If we don't need backwards compatibility in memory layout, these members can of course just go away.

Reply:
backward compatibility to fftw?

Reply:
Not to FFTW, but maybe for interfacing with other languages. I'm thinking about the Matlab interface for example. If you have to describe the memory layout of the FINUFFT plan type there, then it would not be good if it changes depending on the used FFT library.
Maybe I'm just overly cautious here; should we just remove the entries when ducc is used and see if anything breaks?